.github/workflows/build-preview.yml

```diff
@@ -42,13 +42,21 @@ jobs:
       - name: Install and Build PR Branch 🔧
         # TODO: when we move shieldlib to its own repo, move shieldlib docs CI also
         run: |
-          npm ci --include=dev
+          npm install --include=dev
+          git clone https://github.com/1ec5/maplibre-gl-js.git --branch=complex-text-50 --depth=1
+          cd maplibre-gl-js/
+          npm install --include=dev
+          npm run build-dist
+          cd ../
+          npm link ./maplibre-gl-js/
+          cd shieldlib/
+          npm link ../maplibre-gl-js
+          cd ../
           npm run build
```

Comment:
If you’re wondering why I don’t just do `npm install --save 1ec5/maplibre-gl-js#complex-text-50` and check in the changes to package.js, it seems like it would require more invasive changes to this codebase:

```
> openstreetmap-americana@0.1.0 build:code
> tsx scripts/build

✘ [ERROR] Could not resolve "maplibre-gl/dist/maplibre-gl.css"

    src/bare_americana.js:3:7:
      3 │ import "maplibre-gl/dist/maplibre-gl.css";
        ╵        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

  You can mark the path "maplibre-gl/dist/maplibre-gl.css" as external to exclude it from the
  bundle, which will remove this error.

✘ [ERROR] Could not resolve "maplibre-gl"

    src/shieldtest.js:4:28:
      4 │ import * as maplibregl from "maplibre-gl";
        ╵                             ~~~~~~~~~~~~~

  You can mark the path "maplibre-gl" as external to exclude it from the bundle, which will remove
  this error.

✘ [ERROR] Could not resolve "maplibre-gl"

    src/americana.js:9:28:
      9 │ import * as maplibregl from "maplibre-gl";
        ╵                             ~~~~~~~~~~~~~

  You can mark the path "maplibre-gl" as external to exclude it from the bundle, which will remove
  this error.

✘ [ERROR] Could not resolve "maplibre-gl/dist/maplibre-gl.css"

    src/americana.js:10:7:
      10 │ import "maplibre-gl/dist/maplibre-gl.css";
         ╵        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

  You can mark the path "maplibre-gl/dist/maplibre-gl.css" as external to exclude it from the
  bundle, which will remove this error.

✘ [ERROR] Could not resolve "maplibre-gl"

    src/js/map_builder.ts:19:64:
      19 │ import maplibregl, { Map, MapOptions, StyleSpecification } from "maplibre-gl";
         ╵                                                                 ~~~~~~~~~~~~~

  You can mark the path "maplibre-gl" as external to exclude it from the bundle, which will remove
  this error.

✘ [ERROR] Could not resolve "maplibre-gl"

    src/search.js:3:28:
      3 │ import * as maplibregl from "maplibre-gl";
        ╵                             ~~~~~~~~~~~~~

  You can mark the path "maplibre-gl" as external to exclude it from the bundle, which will remove
  this error.

✘ [ERROR] Could not resolve "maplibre-gl"

    src/js/legend_control.js:8:28:
      8 │ import * as maplibregl from "maplibre-gl";
        ╵                             ~~~~~~~~~~~~~

  You can mark the path "maplibre-gl" as external to exclude it from the bundle, which will remove
  this error.


  shieldlib/dist/index.js       32.0kb
  shieldlib/dist/index.js.map  150.9kb

⚡ Done in 12ms
7 errors
ERROR: "build:code" exited with 1.
```